pkg/await/await.go

```diff
@@ -116,6 +142,18 @@ func Creation(
 	case coreV1LimitRange:
 	case coreV1Namespace:
 	case coreV1Secret:
+	case rbacAuthorizationV1ClusterRole:
+	case rbacAuthorizationV1ClusterRoleBinding:
+	case rbacAuthorizationV1Role:
+	case rbacAuthorizationV1RoleBinding:
+	case rbacAuthorizationV1Alpha1ClusterRole:
+	case rbacAuthorizationV1Alpha1ClusterRoleBinding:
+	case rbacAuthorizationV1Alpha1Role:
+	case rbacAuthorizationV1Alpha1RoleBinding:
+	case rbacAuthorizationV1Beta1ClusterRole:
+	case rbacAuthorizationV1Beta1ClusterRoleBinding:
+	case rbacAuthorizationV1Beta1Role:
+	case rbacAuthorizationV1Beta1RoleBinding:
 		break
 
 	// TODO(hausdorff): Find some sensible default for unknown kinds.
```

Comment:
In other repos, we've generally made a point to open issues to track `TODO`s in code.  This may be valuable here as well so we have an enumeration of the remaining areas we have similar work we need to do.  

We've been using the `// TODO[pulumi/pulumi-kubernetes#1234] Find some sensible default for unknown kinds` comment format.

Reply:
Actually this is causing enough problems that we should just fix this now, in the same PR as we re-organize the `switch` statement, which you mention doing below.